webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -328,9 +328,9 @@ export default {
     eventInput() {
       if (this.createEvent) {
         return {
-          eventStart: this.formData.eventStart,
+          eventStart: new Date(this.formData.eventStart).toISOString(),
```

Comment:
How does this change the fact that we saved a timezoned Date?
Does this call ensure that the time ins transformed to UTC?

Reply:
Yes.

Reply:
The browser takes care of it.